github/messages_test.go

```diff
@@ -110,7 +110,7 @@ func TestValidatePayload_FormGet(t *testing.T) {
 
 	// check that if payload is invalid we get error
 	req.Header.Set(signatureHeader, "invalid signature")
-	if _, err = ValidatePayload(req, nil); err == nil {
+	if _, err = ValidatePayload(req, []byte{}); err == nil {
```

Comment:
This and line 143 will now need to be changed... since it is checking for an invalid signature, please just go ahead and use something like `[]byte{0}` here and below.